ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -198,6 +198,13 @@
           + "\"off\" disables the validator. If set to \"auto\", KSQL will attempt to discover "
           + "whether the Kafka cluster supports the required API, and enables the validator if "
           + "it does.";
+  public static final String KSQL_PULL_QUERIES_SKIP_ACCESS_VALIDATOR_CONFIG =
+      "ksql.query.pull.skip.access.validator";
+  public static final boolean KSQL_PULL_QUERIES_SKIP_ACCESS_VALIDATOR_DEFAULT = false;
+  public static final String KSQL_PULL_QUERIES_SKIP_ACCESS_VALIDATOR_DOC = "If true, KSQL will "
+      + " NOT enforce access validation checks for pull queries, which could expose Kafka topics"
```

Comment:
`validation checks` here is a bit vague.  Can we be more explicit?  This is really about skips `authorization checks`.  The docs and the name of this config should reflect this.

I think the actual functionality is that with this set to the default `false` KSQL won't support pull queries when running against a secure Kafka. With it set to `true` KSQL won't check the user has access to the topics underlying the the materialized state the pull query is accessing.

Is that right?